qa/mc_sc_forging4.py

```diff
@@ -69,22 +69,22 @@ def run_test(self):
         # Generate 5 SC blocks
         scblock_ids = generate_next_blocks(sc_node, "first node", 5)
 
-        # Verify that SC block contains newly created MC blocks as MainchainHeaders and MainchainReferences
-        # First 4 SC blocks. Every block contain 3 MainchainReferences and 49 MainchainHeaders
+        # Verify that SC block contains newly created MC blocks as MainchainHeaders and MainchainReferenceData
+        # First 4 SC blocks. Every block contains 49 MainchainHeaders and 49 MainchainReferenceData
         for i in range(4):
             check_mcheaders_amount(49, scblock_ids[i], sc_node)
             for mchash in mcblock_hashes[i * 49 : (i + 1) * 49]:
                 check_mcheader_presence(mchash, scblock_ids[i], sc_node)
-            check_mcreferencedata_amount(3, scblock_ids[i], sc_node)
-            for mchash in mcblock_hashes[i * 3 : (i + 1) * 3]:
+            check_mcreferencedata_amount(49, scblock_ids[i], sc_node)
```

Comment:
The test will pass as soon as MC websocket optimization PR will be merged: https://github.com/HorizenOfficial/zend_oo/pull/171

Currently, we have not enough time to forge block with 49 McBlockRefData entries.